examples/google-cloud-speech/output/google-cloud-speech/lib/google/cloud/speech/v1/speech_client.rb

```diff
@@ -0,0 +1,411 @@
+              updater_proc: updater_proc,
+              scopes: scopes,
+              interceptors: interceptors,
+              &Google::Cloud::Speech::V1::Speech::Stub.method(:new)
+            )
+
+            @recognize = Google::Gax.create_api_call(
+              @speech_stub.method(:recognize),
+              defaults["recognize"],
+              exception_transformer: exception_transformer
+            )
+            @long_running_recognize = Google::Gax.create_api_call(
+              @speech_stub.method(:long_running_recognize),
+              defaults["long_running_recognize"],
+              exception_transformer: exception_transformer
+            )
+            @streaming_recognize = Google::Gax.create_api_call(
+              @speech_stub.method(:streaming_recognize),
+              defaults["streaming_recognize"],
+              exception_transformer: exception_transformer
+            )
+          end
+
+          # Service calls
+
+          # Performs synchronous speech recognition: receive results after all audio
+          # has been sent and processed.
+          #
+          # @param config [Google::Cloud::Speech::V1::RecognitionConfig | Hash]
+          #   *Required* Provides information to the recognizer that specifies how to
+          #   process the request.
+          #   A hash of the same form as `Google::Cloud::Speech::V1::RecognitionConfig`
+          #   can also be provided.
+          # @param audio [Google::Cloud::Speech::V1::RecognitionAudio | Hash]
+          #   *Required* The audio data to be recognized.
+          #   A hash of the same form as `Google::Cloud::Speech::V1::RecognitionAudio`
+          #   can also be provided.
+          # @param options [Google::Gax::CallOptions]
+          #   Overrides the default settings for this call, e.g, timeout,
+          #   retries, etc.
+          # @yield [result, operation] Access the result along with the RPC operation
+          # @yieldparam result [Google::Cloud::Speech::V1::RecognizeResponse]
+          # @yieldparam operation [GRPC::ActiveCall::Operation]
+          # @return [Google::Cloud::Speech::V1::RecognizeResponse]
+          # @raise [Google::Gax::GaxError] if the RPC is aborted.
+          # @example
+          #   require "google/cloud/speech"
+          #
+          #   speech_client = Google::Cloud::Speech.new(version: :v1)
+          #   encoding = :FLAC
+          #   sample_rate_hertz = 44100
+          #   language_code = "en-US"
+          #   config = {
+          #     encoding: encoding,
+          #     sample_rate_hertz: sample_rate_hertz,
+          #     language_code: language_code
+          #   }
+          #   uri = "gs://bucket_name/file_name.flac"
+          #   audio = { uri: uri }
+          #   response = speech_client.recognize(config, audio)
+
+          def recognize \
+              config,
+              audio,
+              options: nil,
+              &block
+            req = {
```

Comment:
The seems like there is a tendency to prefer full names - should we change this to request?

Reply:
Depending on where we land in terms of line lengths and rubocop, yes.

Reply:
Changed to `request` (and `requests`).